seeds/addresses_events/first_funded_by_seed.csv

```diff
@@ -2,7 +2,7 @@ blockchain,address,first_funded_by,block_number,tx_hash
 gnosis,0x000000000000c1a8cc407293a22bff068d4584d2,0xbe51d38547992293c89cc589105784ab60b004a9,25390104,0x80f6ddbb341615b5ed83a2d1a3e51deb202edfffa24dd6a787c245db68bb6ed5
 gnosis,0x000000a52a03835517e9d193b3c27626e1bc96b1,0x115909bdcbab21954beb4ab65fc2abee9866fa93,21134610,0x75e2c7811c9c87def2fc113d5b01bf51e100fc449bda4cbdcec66d3b1215bd07
 goerli,0x1a32f88ca270e1092dac040ce6cf629fbc60d1b8,0xe73f10e30ba43e55dcb9844e9a91fc1c42ad0fea,6398877,0x92b47d7ef3dad447885d066e83851e7b3cbd0a9eef44c22d0d8dff3f5611631b
-goerli,0x1a332b3cf20466189c221a04f92e47c21a17ba19,0x5cf0e1f993131e12c5fdd3dfc8ba52b30fcdcc14,4449324,0x99866f8bb5e22538a11a67afc11cdae62e3e9c00506e9f8642f5819d1d5ec9e1
+goerli,0x1a332b3cf20466189c221a04f92e47c21a17ba19,0x5cf0e1f993131e12c5fdd3dfc8ba52b30fcdcc14,4449324,0xd64697d0f8b3455a40385b6d71170ee5fd65f7f7e833c3d2782eccaec10d809e
```

Comment:
Based on etherscan, there are 2 tx_hash for this same block_number:
https://goerli.etherscan.io/address/0x1a332b3cf20466189c221a04f92e47c21a17ba19

Reply:
And `0xd64697d0f8b3455a40385b6d71170ee5fd65f7f7e833c3d2782eccaec10d809e` may be the earlier transaction.